src/beamlime/config/raw_detectors/dream.py

```diff
@@ -53,9 +52,8 @@
                 },
                 transpose=('wire', 'module', 'segment', 'counter', 'strip'),
                 select={'wire': 0},
-                flatten={'z_id': ('module', 'segment', 'counter')},
+                flatten={'mod/seg/cntr': ('module', 'segment', 'counter')},
```

Comment:
Unrelated: `z` was misleading, since it was "up", whereas `z` in the instrument coordinates is actually along the beam (`y` would be "up").